aws/resource_aws_worklink_website_certificate_authority_association.go

```diff
@@ -0,0 +1,170 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/worklink"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+func resourceAwsWorkLinkWebsiteCertificateAuthorityAssociation() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsWorkLinkWebsiteCertificateAuthorityAssociationCreate,
+		Read:   resourceAwsWorkLinkWebsiteCertificateAuthorityAssociationRead,
+		Delete: resourceAwsWorkLinkWebsiteCertificateAuthorityAssociationDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"fleet_arn": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"certificate": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"display_name": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringLenBetween(0, 100),
+			},
+			"website_ca_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceAwsWorkLinkWebsiteCertificateAuthorityAssociationCreate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).worklinkconn
+
+	input := &worklink.AssociateWebsiteCertificateAuthorityInput{
+		FleetArn:    aws.String(d.Get("fleet_arn").(string)),
+		Certificate: aws.String(d.Get("certificate").(string)),
+	}
+
+	if v, ok := d.GetOk("display_name"); ok {
+		input.DisplayName = aws.String(v.(string))
+	}
+
+	resp, err := conn.AssociateWebsiteCertificateAuthority(input)
+	if err != nil {
+		return fmt.Errorf("Error creating WorkLink Website Certificate Authority Association: %s", err)
+	}
+
+	d.SetId(fmt.Sprintf("%s/%s", aws.StringValue(resp.WebsiteCaId), d.Get("fleet_arn").(string)))
```

Comment:
Two things here:
* I'm not sure if we note this in our contributing documentation anywhere (😅), but by convention across the Terraform AWS provider we prefer to set resource IDs (and therefore their import IDs) where the container ID is first, then the contained ID. So in this case WorkLink Fleet ARN then WorkLink Website CA ID. 👍 
* We may want to consider a different separator, e.g. a comma, since the ARN involved contains slashes

```suggestion
	d.SetId(fmt.Sprintf("%s,%s", d.Get("fleet_arn").(string), aws.StringValue(resp.WebsiteCaId)))
```

Reply:
Thanks. I fixed to given format.